ansible/roles/yarn/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Node
+- name: Register NodeJS
+  shell: 'curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -'
+  changed_when: false
+
+- name: Remove cmdtest
+  apt:
+    name: cmdtest
+    state: absent
+
+- name: Install Yarn APT key
+  apt_key:
+    data: "{{ lookup('file', 'pubkey.gpg') }}"
+    state: present
+
+- name: Add yarn repository
+  apt_repository:
+    repo: 'deb https://dl.yarnpkg.com/debian/ stable main'
+    update_cache: yes
+
+- name: Install packages
+  apt:
+    name: '{{ item }}'
+    state: latest
```

Comment:
I know we have used this elsewhere but we should avoid `state: latest` where possible. It results in updated versions being installed as soon as available. `present` is safer.

Reply:
Done.